demo/lightning/image-segmentation/README.md

```diff
@@ -0,0 +1,167 @@
+
+    Defines the `DatafluxPytTrain` class which wraps two `DataFluxMapStyleDataset`s corresponding to the `images` and the `labels` in the raw dataset.
+    ```py
+    def __init__(
+            self,
+            project_name,
+            bucket_name,
+            **kwargs,
+        ):
+
+            self.train_transforms = kwargs["transforms"]
+            patch_size, oversampling = kwargs["patch_size"], kwargs["oversampling"]
+            self.patch_size = patch_size
+            self.rand_crop = RandBalancedCrop(patch_size=patch_size,
+                                            oversampling=oversampling)
+
+            # Dataflux-specific setup.
+            self.project_name = project_name
+            self.bucket_name = bucket_name
+
+            self.images_dataset = dataflux_mapstyle_dataset.DataFluxMapStyleDataset(
+                project_name=self.project_name,
+                bucket_name=self.bucket_name,
+                config=dataflux_mapstyle_dataset.Config(
+                    # This needs to be True to map images with labels
+                    sort_listing_results=True,
+                    prefix=kwargs["images_prefix"],
+                ),
+            )
+
+            self.labels_dataset = dataflux_mapstyle_dataset.DataFluxMapStyleDataset(
+                project_name=self.project_name,
+                bucket_name=self.bucket_name,
+                config=dataflux_mapstyle_dataset.Config(
+                    # This needs to be True to map images with labels
+                    sort_listing_results=True,
+                    prefix=kwargs["labels_prefix"],
+                ),
+            )
+    ```
+
+1. `data.py`
+
+    Implements the `Unet3DDataModule` class which inherits from PyTorch Lightning's `LightningDataModule`. An instance of the `DatafluxPytTrain` class is created in its `setup` function.
+
+    ```py
+    def setup(self, stage="fit"):
+        if stage == "fit":
+            train_data_kwargs = {
+                "patch_size": self.args.input_shape,
+                "oversampling": self.args.oversampling,
+                "seed": self.args.seed,
+                "images_prefix": self.args.images_prefix,
+                "labels_prefix": self.args.labels_prefix,
+                "transforms": get_train_transforms(),
+            }
+            self.train_dataset = DatafluxPytTrain(
+                project_name=self.args.gcp_project,
+                bucket_name=self.args.gcs_bucket,
+                **train_data_kwargs,
+            )
+            self.train_sampler = None
+            if self.args.num_workers > 1:
+                self.train_sampler = DistributedSampler(
+                    self.train_dataset,
+                    seed=self.args.seed,
+                    drop_last=True
+                )
+    
+    ```
+
+1. `train.py`
+
+    An instance of the `Unet3DDataModule` class is created and passed to the Pytorch Lightning trainer instance's `fit` function as the `train_dataloaders`
```

Comment:
"in the train_dataloader keyword argument"

Reply:
Done.